test/jit/test_backends.py

```diff
@@ -562,6 +562,33 @@ def test_execution(self):
         input = torch.randn(5)
         self.check_forward((input, input))
 
+class ErrorMessagesWithCompiler(JitBackendTestCase):
+    """
+    Tests for errors that occur with compiler, specifically:
+        * an operator is not supported by the backend
+    """
+
+    class ModuleNotSupported(torch.nn.Module):
+        """
+        A module with an operator that is not supported.
+        """
+        def __init__(self):
+            super().__init__()
+
+        def forward(self, x, h):
+            return x * h
+            self._loweredmodule.forward()
+
+    def setUp(self):
+        super().setUp()
+
+    def test_errors(self):
+        # Test exception is thrown.
+        with self.assertRaisesRegexWithHighlight(RuntimeError,
+                                                 r"node of aten::mul is not supported in this compiler", ""):
+            scripted_module_n = torch.jit.script(ErrorMessagesWithCompiler.ModuleNotSupported())
```

Comment:
This line can be out of `with self.assertRaisesRegexWithHighlight(...`, so that readers know that the error is from the line of  `_jit_to_backend`.